app/core/services/tts_cartesia.py

```
@@ -65,6 +77,33 @@ def get_audio_format(self) -> str:
+        return self._ws
+
+    async def _reset_ws(self):
+        """연결을 폐기합니다. 오류 발생 시 호출합니다."""
+        async with self._ws_lock:
+            ws = self._ws
+            self._ws = None
+        if ws is not None:
+            try:
+                await ws.close()
+            except Exception:
+                pass
```

Ruff (0.15.13)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)